contribs/gmf/src/source/asitvdsource.js

```diff
@@ -0,0 +1,64 @@
+goog.provide('gmf.source.AsitVD');
+
+goog.require('ol.Attribution');
+goog.require('ol.source.WMTS');
+goog.require('ol.tilegrid.WMTS');
+
+
+/**
+ * @const {!Array.<number>}
+ */
+gmf.source.AsitVDResolutions = [
+  4000, 3750, 3500, 3250, 3000, 2750, 2500, 2250, 2000, 1750, 1500, 1250,
+  1000, 750, 650, 500, 250, 100, 50, 20, 10, 5, 2.5, 2, 1.5, 1, 0.5
+];
+
+
+/**
+ * @const {ol.tilegrid.WMTS}
+ */
+gmf.source.AsitVDTileGrid = new ol.tilegrid.WMTS({
+  extent: [420000, 30000, 900000, 350000],
+  resolutions: gmf.source.AsitVDResolutions,
+  matrixIds: gmf.source.AsitVDResolutions.map(function(value, index) {
+    return String(index);
+  })
+});
+
+
+/**
+ * @typedef {{layer: string}}
+ */
+gmf.source.AsitVDOptions;
+
+
+
+/**
+ * @constructor
+ * @extends {ol.source.WMTS}
+ * @param {gmf.source.AsitVDOptions} options WMTS options.
+ */
+gmf.source.AsitVD = function(options) {
```

Comment:
The won't work when ngeo.js is used. The options should be defined in the `ngeox` namespace.